source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -739,25 +743,45 @@ this parameter is ignored.
 
 **interval**
 
-Specifies the elapsed time between every inventory synchronization.
+Specifies the initial number of seconds between every inventory synchronization. If synchronization fails the value will be duplicated until it reaches the value of ``max_interval``.
 
 +--------------------+----------------------------------------------------------------------+
-| **Default value**  | 5 m                                                                |
+| **Default value**  | 5 m                                                                  |
 +--------------------+----------------------------------------------------------------------+
 | **Allowed values** | Any number greater than or equal to 0. Allowed sufixes (s, m, h, d). |
 +--------------------+----------------------------------------------------------------------+
 
-**min_interval**
+**response_timeout**
 
-Defines the minimum interval between two synchronizations. A new sync process is blocked if less than ``min_interval``
-seconds have elapsed since the last sync message was sent by the agent.
+Specifies the time elapsed in seconds since the agent sends the message to the manager and receives the response.
+If the response is not received in this interval, the message is marked as unanswered (timed-out) and the agent may start a new synchronization session at the defined interval.
 
 +--------------------+----------------------------------------------------------------------+
-| **Default value**  | 1 m                                                                  |
+| **Default value**  | 30 s                                                                 |
```

Comment:
If the definition will be always seconds we have to remove the `s` and the suffixes in the `Allowed values` file.